src/DynamoCore/Rendering/DefaultRenderPackageFactory.cs

```diff
@@ -42,7 +42,8 @@ public class DefaultRenderPackage : IRenderPackage
         private List<int> meshIndices = new List<int>();
  
         private List<int> lineStripVertexCounts = new List<int>();
- 
+        private byte[] colors;
```

Comment:
These are updates to our `DefaultRenderPackageFactory` to support storing arrays of texture color data in the render package.